app/models/user.rb

```diff
@@ -1,3 +1,4 @@
 class User < ApplicationRecord
+  validates :email, presence: true, uniqueness: true
```

Comment:
added validations for user info - validation testing covered in user feature tests